libs/api/domains/ministry-of-justice/src/lib/ministryOfJustice.resolver.ts

```diff
@@ -0,0 +1,34 @@
+import type { User } from '@island.is/auth-nest-tools'
+import {
+  CurrentUser,
+  IdsAuthGuard,
+  IdsUserGuard,
+  Scopes,
+} from '@island.is/auth-nest-tools'
+import { ApiScope } from '@island.is/auth/scopes'
+import { Audit } from '@island.is/nest/audit'
+import { UseGuards } from '@nestjs/common'
+import { Args, Query, Resolver } from '@nestjs/graphql'
+import { MinistryOfJusticeService } from './ministryOfJustice.service'
+import { SearchCaseTemplateInput } from './models/SearchCaseTemplate.input'
+import { SearchCaseTemplateResponse } from './models/SearchCaseTemplate.response'
+
+@UseGuards(IdsUserGuard, IdsAuthGuard)
```

Comment:
You don't need both the guards. `IdsUserGuard` suffice, and that guard forces the `nationalId` claim to be present in the access token, to force end user authentication. While `IdsAuthGuard` requires a valid access token where the `nationalId` claim is optional, so it allows machine client tokens, which we don't want here.